services/ui-gateway/src/browser/pages/web/WebPage.tsx

```diff
@@ -56,11 +56,11 @@ const Layout = () => {
         <Box
             component="main"
             sx={{
-                minHeight: '100%',
+                minHeight: '100vh',
```

Comment:
Adding `hight: 100%` would also do the trick and avoiding view-height issues. Only having `min-height` is not enough to get a div to fill 100% of it's parent.